scopes/preview/preview/preview.main.runtime.tsx

```diff
@@ -659,7 +663,14 @@ export class PreviewMain {
     const [name, uiRoot] = this.getUi();
     const resolvedAspects = await this.resolveAspects(PreviewRuntime.name, undefined, uiRoot);
     const filteredAspects = this.filterAspectsByExecutionContext(resolvedAspects, context, aspectsIdsToNotFilterOut);
-    const filePath = await this.ui.generateRoot(filteredAspects, name, 'preview', PreviewAspect.id);
+    const filePath = await this.ui.generateRoot(
```

Comment:
To generate the root file into `node_modules/.cache` from the package `@teambit/ui` in bit CLI, I have to get a custom path that points to the workspace root. So I passed it into every `ui.generateRoot()` call.